pkg/cmd/server/api/types.go

```diff
@@ -998,3 +1001,26 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string
 }
+
+// ClusterResourceOverrideConfig is the configuration for the ClusterResourceOverride
+// admission controller which overrides user-provided container request/limit values.
+type ClusterResourceOverrideConfig struct {
+	unversioned.TypeMeta
+	// Enabled must be true for the plugin to do anything.
+	// The plugin's actions can be disabled per-project with the project annotation
+	// quota.openshift.io/cluster-resource-override-enabled="false", so cluster admins
+	// can exempt infrastructure projects and such from the overrides.
+	Enabled bool
```

Comment:
Thinking about it some more, @deads2k @liggitt I don't like the absence of config to mean that the plugin is disabled. That would mean that an empty config vs a nil config mean totally different things. My preference would be that an empty config mean the same thing as a nil config.